src/ToricVarieties/LineBundles/properties.jl

```diff
@@ -0,0 +1,73 @@
+@doc Markdown.doc"""
+    istrivial(l::ToricLineBundle)
+
+Returns "true" if the toric line bundle `l` is trivial and "false" otherwise.
+
+# Examples
+```jldoctest
+julia> H = hirzebruch_surface(4)
+A normal, non-affine, smooth, projective, gorenstein, q-gorenstein, non-fano, 2-dimensional toric variety without torusfactor
+
+julia> istrivial(ToricLineBundle(H, [1,0]))
+false
+
+julia> istrivial(ToricLineBundle(H, [0,0]))
+true
+```
+"""
+istrivial(l::ToricLineBundle) = isprincipal(toric_divisor(l))
+export istrivial
+
+
+@doc Markdown.doc"""
+    isbasepoint_free(l::ToricLineBundle)
```

Comment:
```suggestion
    is_basepoint_free(l::ToricLineBundle)
```

Reply:
Thank you @fingolfin. Let me adjust this.